pkgs/top-level/all-packages.nix

```diff
@@ -1508,6 +1508,8 @@ with pkgs;
 
   font-v = with python3Packages; toPythonApplication font-v;
 
+  fontbakery = with python3Packages; toPythonApplication fontbakery;
```

Comment:
Is this package supposed to be available as a python module as well?

Reply:
It required as a module for https://github.com/notofonts/notobuilder/ which we would need if we ever want to build noto-fonts from source

Reply:
All the packages in this PR should be available as Python modules (since a lot of font build processes will depend on them that way, even if they're really a command line tool like fontbakery). Plus for all those packages where upstream's docs give some indication that the tool is designed to be invoked from the command line, I've exported them as Python applications as well.